webpack/components/Content/Details/__tests__/__snapshots__/ContentDetails.test.js.snap

```diff
@@ -7,7 +7,7 @@ exports[`Content Details Info should render and contain appropriate components 1
     loadingText="Loading"
     timeout={300}
   >
-    <Uncontrolled(TabContainer)
+    <ForwardRef
```

Comment:
I think this is unrelated to the PR as well as the other snap updates

Reply:
yes, hopefully this will be gone by the time this is rebased and ready to merge.

Reply:
update: gone :)